README.md

```diff
@@ -30,15 +30,15 @@ To get started with using Teleporter, see [How to Deploy Teleporter Enabled Subn
 - [Docs](#docs)
 - [Resources](#resources)
 
-## Deployed Addresses
+## Deployed Addresses for Latest Major Version
 
 | Contract              | Address                                        | Chain                    |
 | --------------------- | ---------------------------------------------- | ------------------------ |
 | `TeleporterMessenger` | **0x253b2784c75e510dD0fF1da844684a1aC0aa5fcf** | All chains, all networks |
 | `TeleporterRegistry`  | **0x7C43605E14F391720e1b37E49C78C4b03A488d98** | Mainnet C-Chain          |
 | `TeleporterRegistry`  | **0xF86Cb19Ad8405AEFa7d09C778215D2Cb6eBfB228** | Fuji C-Chain             |
 
-- When deployed using [Nick's method](https://yamenmerhi.medium.com/nicks-method-ethereum-keyless-execution-168a6659479c#), `TeleporterMessenger` will be at the same address on all chains. See [Teleporter Contract Deployment](./utils/contract-deployment/README.md) and [Deploy Teleporter to a Subnet](#deploy-teleporter-to-a-subnet) for more details.
+- Using [Nick's method](https://yamenmerhi.medium.com/nicks-method-ethereum-keyless-execution-168a6659479c#), `TeleporterMessenger` deploys at a universal address across all chains, varying with each teleporter major release. **Compatibility exists only between same-version `TeleporterMessenger` instances.** See [Teleporter Contract Deployment](./utils/contract-deployment/README.md) and [Deploy Teleporter to a Subnet](#deploy-teleporter-to-a-subnet) for more details.
```

Comment:
There was a mention of different "major" releases of Teleporter in the first sentence. Is compatibility provided only between the exact same version of TeleporterMessenger instances, or are instances compatible as long as they are part of the same major release (but differ in their minor release/version)?

Reply:
Technically speaking, any change to the `TeleporterMessenger` bytecode creates an incompatibility. This is reflected in our release/versioning policy of reserving major releases for such changes. Minor/Patch releases should remain compatible. To ensure this, we reserve such releases for contract changes that do not change the `TeleporterMessenger` bytecode, and for non-Solidity utility changes.

I've gone ahead and added this to the README as well.